lib/content-services/src/lib/content-node-selector/content-node-selector-panel.component.ts

```diff
@@ -194,7 +198,7 @@ export class ContentNodeSelectorPanelComponent implements OnInit, OnDestroy {
     showingSearchResults: boolean = false;
     loadingSearchResults: boolean = false;
     inDialog: boolean = false;
-    _chosenNode: Node = null;
+    _chosenNode: Node [] = null;
```

Comment:
I think underscore is not needed here

Reply:
yep it is needed because we have the    `get chosenNode() {`